stdlib/3/ast.pyi

```diff
@@ -14,9 +14,21 @@ from typing_extensions import Literal
 from _ast import *  # type: ignore
 
 if sys.version_info >= (3, 8):
-    class Num(Constant): ...
-    class Str(Constant): ...
-    class Bytes(Constant): ...
+    class Num(Constant):
+        value: complex
+        kind: Optional[str]
+        # Aliases for value, for backwards compatibility
+        s: Any
```

Comment:
This is `n`, not `s`

Reply:
`n` is inherited from `Constant` as a `complex` type already, so is `s` as `Any` type, I will remove the declaration